src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -548,6 +548,15 @@ export class Connector implements IConnector {
                         await this.getFavicon((await this._dom.querySelectorAll('link[rel~="icon"]'))[0]);
                     }
 
+                    await new Promise((resolve) => {
+                        Tracing.tracingComplete(() => {
+                            this._server.emitAsync('tracing::end', event);
+                            resolve();
+
+                        });
+                        debug(`Ending Collecting Chrome traces`);
+                        Tracing.end();
```

Comment:
I think Tracing.end would throw just as Page.navigate or other chrome commands would. This just sends the command over to chrome over the protocol, so IMHO, it should be fine.